portfolio/src/main/webapp/profile.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<!--Menu -->
+<nav class="menu-main">
+  <a href="index.html">Index</a>
+  <a href="profile.html" class="active">Profile</a>
+  <a href="random.html">Gallery</a>
+  <a href="sps.html">SPS Experience</a>
+  <a href="instagram.html">My instagram</a>
+</nav>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Vanesa  | Profile </title>
+    <link rel="shortcut icon" type="image/jpg" href="images/favicon-32x32.jpg"> 
+    <link rel="stylesheet" href="style.css" >
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+          <p style="background-color:pink;"><b>About me </b></pstyle>
```

Comment:
It's generally easier to change the style later if you use css classes for styling. The web has an aspirational goal of separating content from style.